mpv.py

```diff
@@ -1814,6 +1845,17 @@ def option_info(self, name):
         except AttributeError:
             return None
 
+    def _register_async_callback(self, callback):
+        if callback is None:
+            key = 18_446_744_073_709_551_615
```

Comment:
Please factor this and the large number below out into class constants, and please write ```2**64-1``` and ```2**64-2``` respectively for readability (python has arbitrary integer precision anyway). Also, for now we still officially support python 3.5, and underscore literals are a 3.6 feature (though I may just drop 3.5/3.6 support very soon as even debian stable is up to 3.7 now).

Reply:
Done in latest force-push